### PR TITLE
documentation: Add README intro from Moore

### PR DESCRIPTION
**Pull request by djohnson-spideroak**

### CODE REVIEW

**Review by mooreoak — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ Aranya is a software development tool for governing access to data and services
 
 Aranya has been designed with an emphasis on security, efficiency, and portability.
 
+The root cause of cyber in-security is complexity; and yet when we attempt to protect our systems our solution is to add more.
```
Comment (ycarmy) [inline README.md:7]: I think the following was intended(?):
```suggestion
The root cause of cyber insecurity is complexity; and yet when we attempt to protect our systems, our solution is to add more.
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ Aranya is a software development tool for governing access to data and services
...
+ 
+Software developers must not expect customers to mitigate defects using external security tools and an endless cycle of patching. Software must become secure by design.
+ 
+Aranya is our contribution to this effort. It is a batteries-included tool which allows developers to produce software with built in mrico-segmentation. This complete solution covers user onboarding, access management, authentication, and authorization, freeing the developer to focus on the problem they which to solve.
```
Comment (ycarmy) [inline README.md:11]:
```suggestion
Aranya is our contribution to this effort. It is a batteries-included tool which allows developers to produce software with built-in micro-segmentation. This complete solution covers access management with user onboarding, authentication and authorization, freeing the developer to focus on the problem they wish to solve.
```

File: README.md [hunk 3]
```diff
@@ -4,6 +4,14 @@ Aranya is a software development tool for governing access to data and services
...
+ 
+For users, software build on Aranya is less complex to operate securely, and is secure regardless of the network it is run on.
```
Comment (ycarmy) [inline README.md:13]:
```suggestion
For users, software built on Aranya is less complex to operate securely, and is secure regardless of the network it is run on.
```

**Review by ycarmy — APPROVED** [review]
lgtm!

**Review by ycarmy — APPROVED** [review]
Thank you, Declan!

**Comment by jdygert-spok** [comment]
Opened internally.

**Pull request closed without merging**